spec/controllers/events_controller_spec.rb

```diff
@@ -170,6 +170,13 @@
         expect(assigns(:email)).to have_attributes(hide_recipients: false, recipients: @event.email_adresses_of_rejected_applicants, reply_to: 'workshop.portal@hpi.de', subject: '', content: '')
       end
     end
+
+    describe "GET #accept_all_applicants" do
+      it "should redirect to the event" do
+        get :accept_all_applicants, id: @event.to_param, session: valid_session
+        expect(response).to redirect_to(@event)
```

Comment:
I think it is all that should be testet here?

What it does: it calls a function (event accept all...) and redirects
And the function functionality is tested elsewhere